src/data/J0.dat

```diff
@@ -0,0 +1,97 @@
+Ion	A	a	B	b	C	c	D

```

Comment:
I think we move these two files to `Sunny.jl/assets/data/form_factor_J0.dat` and `form_factor_J2.dat`

Reply:
Ok.